ode/awa.tex

```diff
@@ -616,7 +616,7 @@ \section{Well-posedness of the IVP}
   $u^{(0)}:I\to\Omega$. From the boundedness of $f$, we obtain for
   $t-t_0 \le T$
   \begin{align*}
-    \abs{u^{(k+1)}(t)-u_0} = \abs{\int_{t_0}^t f(s,u^{(k)}(s))\ds} \le \int_{t_0}^t
+    \abs{u^{(k+1)}(t)-u_0} = \left|\int_{t_0}^t f(s,u^{(k)}(s))\ds \right| \le \int_{t_0}^t
```

Comment:
DA sollte lieber \abs umdefiniert werden, wenn das kein \left und \right enthält

Reply:
Gerade nachgeguckt, steht sogar schon drin (im mathsim.osty). Ich habe es jetzt in \abs{} umgeändert und gucke es mir, sollte das weiter nicht funktionieren, nochmal an.

Reply:
Ah, ich sehe, es ist nicht im .osty, sondern im mathsim.sty ! Da steht es als  \DeclarePairedDelimiter{\abs}{\lvert}{\rvert}. Ich habe zwei Ideen: Ich könnte es im mathsim.sty so ändern, wie es auch im mathsim.osty steht: \def\abs#1{\left|#1\right|}.
Oder ich kann \abs[big]{...} setzen lassen, das sollte auch gut gehen, zumindest sieht das bei mir sehr gut aus. :-) Ist nur eine lokale Lösung, dafür fühle ich mich damit wohler, als an Ihrem Paket rumzuspielen.